src/twisted/application/twist/test/test_twist.py

```diff
@@ -201,3 +201,102 @@ def run(self):
             )
         )
         self.assertEqual(runners[0].runs, 1)
+
+
+
+class CrippledRunner(object):
```

Comment:
No problem here.  Btw, I used it in a similar context to where it is used elsewhere in the code (see test_twistd).